Caltech-IPAC/firefly
# IRSA-253: create a hydra template as a starting point to migrate FinderChart

https://caltech-ipac.atlassian.net/browse/IRSA-253

A placeholder app for the new migrated FinderChart.  This pull request includes a base implementation of the Hydra template.
To Test:
- check out 'IRSA-253_hydra_template' from both firefly and ife
- in ife, build finderchart, ie.  gradle :finderchart:buildAndDeploy
- on your browser, goto http://localhost:8080/applications/finderchart/
